content/rc/api/get-started/use-rest-api.md

```diff
@@ -12,11 +12,15 @@ aliases: /rv/api/how-to/using-curl/
          /rc/api/get-started/use-rest-api.md         
 ---
 You can access and use the API endpoint URI ([`https://api.redislabs.com/v1`](https://api.redislabs.com/v1/)) with any of the following tools:
-
+ß
```

Comment:
Was this supposed to be here?

Reply:
Nope! Removed it in [cece8ce](https://github.com/RedisLabs/redislabs-docs/pull/3112/commits/cece8cea74405e5783c3a11693d5415352026915).